tests/core/test_message.c

```diff
@@ -190,7 +190,8 @@ void test_message_with_tx_serialize() {
   TEST_ASSERT(tx_essence_add_output(essence, OUTPUT_EXTENDED, extended_output_two) == 0);
 
   // add tagged data payload
-  tagged_data_t* tagged_data = tagged_data_create("HORNET FAUCET", NULL, 0);
+  char const* const hornet_faucet = "HORNET FAUCET";
+  tagged_data_t* tagged_data = tagged_data_create((byte_t*)hornet_faucet, strlen(hornet_faucet), NULL, 0);
```

Comment:
should include the byte of the null terminator
```suggestion
  tagged_data_t* tagged_data = tagged_data_create((byte_t*)hornet_faucet, strlen(hornet_faucet)+1, NULL, 0);
```

Reply:
Done.